commands/initial/templates/tasks/test.js

```diff
@@ -13,45 +13,40 @@ function run(type) {
     server.start();
 }
 
-function getConfig(type) {
-    switch (type) {
-        case 'headless':
-        case 'hl':
-        case 'h':
-            return getHeadlessConfig();
-        case 'all':
-        case 'a':
-            return getAllConfig();
-        case 'watch':
-        case 'w':
-            return getWatchConfig();
-        default:
-            return getSingleConfig();
-    }
-}
-
-function getSingleConfig() {
-    let config = getHeadlessConfig();
-
-    config.singleRun = true;
+function getConfig(options) {
+	let config = getAllConfig(options.watch);
+	config.browsers = options.browsers;
+	config.singleRun = !options.watch;
 
-    return config;
+	return config;
 }
 
-function getHeadlessConfig() {
-    let config = getAllConfig();
-
-    config.browsers = ['PhantomJS'];
-
-    return config;
-}
-
-function getWatchConfig() {
-    let config = getAllConfig(true);
-
-    config.browsers = ['Chrome'];
-
-    return config;
+function parseOptions(args){
+	let options = {};
+
+	args.forEach((arg, index)=>{
```

Comment:
The changes that would help here are to just follow the coding conventions/style I've been using.

1. Use `args.reduce((options, arg) => { /* ...code... */ }, {});` instead of a `.forEach`
2. Single quotes instead of double quotes
3. Use `const` instead of `let` for variables that won't change

Admittedly, I need to do a better job of managing the styling via ESLint or Prettier, but I never got around to it